waspc/waspls/src/Wasp/LSP/Syntax.hs

```diff
@@ -28,6 +31,14 @@ lspPositionToOffset srcString (J.Position l c) =
    in -- We add 1 to the length of each line to make sure to count the newline
       sum (map ((+ 1) . length) linesBefore) + fromIntegral c
 
+-- | @lspRangeToSpan srcString range@ returns 0-based source span from start of
```

Comment:
"To the specified line and column" -> but the range is not just one (line, column) pair, it is two of them, right? So how does that work then? What is the relationship between the two inputs, srcString, and Range?